generators/app/tasks/appSetup/androidProjectSetup.js

```diff
@@ -16,6 +16,18 @@ function addDotenvPluginAndRNScreen() {
     'if (enableHermes) {',
     "implementation 'androidx.appcompat:appcompat:1.2.0-alpha03'\n\timplementation 'androidx.multidex:multidex:2.0.1'\n\tif (enableHermes) {"
   );
+  updatedBuildGradleContent = updatedBuildGradleContent.replace(
+    'compileSdkVersion rootProject.ext.compileSdkVersion',
+    'compileSdkVersion rootProject.ext.compileSdkVersion\nflavorDimensions "buildtype"'
+  );
+  updatedBuildGradleContent = updatedBuildGradleContent.replace(
+    '// applicationVariants are e.g. debug, release',
+    'productFlavors {\nqa {\ndimension \'buildtype\'\napplicationIdSuffix ".qa"\n}\nstage {\ndimension \'buildtype\'\napplicationIdSuffix ".stage"\n}\nproduction {\ndimension \'buildtype\'\napplicationIdSuffix ".production"\n}\n}\n// applicationVariants are e.g. debug, release'
+  );
```

Comment:
Same. Also, the production env doesn't have the `.production` in the end, remove it
<img width="529" alt="Captura de Pantalla 2020-04-02 a la(s) 12 58 59" src="https://user-images.githubusercontent.com/8583214/78275937-af452380-74e8-11ea-8c7d-48fe24bf2a10.png">